macos/CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Fluent UI for macOS
+
+## Demo app
+
+Included in this repository is a demo app of currently showcase all the available Fluent controls.

```

Comment:
The previous CONTRIBUTING.md file has this as its first line: "Included in this repository is a demo of currently implemented controls."

Should they be consistent? If not, this one should probably be re-written to be something like "Included in this repository is a demo app showcasing all of the currently available Fluent controls."